index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>UI card</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="card">
+        <div class="imgbx">
+          <img src="NikeShoes.png" alt="NikeShoes" />
+        </div>
+        <div class="contentbx">
+          <h2>Nike Shoes</h2>
+          <div class="size">
+            <h3>size:</h3>
+            <span>7</span>
+            <span>8</span>
+            <span>9</span>
+            <span>10</span>
+          </div>
+          <div class="color">
+            <h3>color:</h3>
+            <span id="color-1"></span>
+            <span id="color-2"></span>
+            <span id="color-3"></span>
+          </div>
+          <a
+            href="https://www.nike.com/in/t/court-vision-low-next-nature-shoes-N2fFHb/DH2987-101"
+            target="#"
```

Comment:
Something exists like this target="#"

Reply:
Yes it does exist and I've used it many times. As i don't want the user to launch the link provided in my anchor tag on the same page i want the link to launch on new tab

Reply:
I don't know about this # but for link to open in new tab we use "_blank" https://www.w3schools.com/tags/att_a_target.asp